js/HotgridView.js

```diff
@@ -0,0 +1,87 @@
+import Adapt from 'core/js/adapt';
+import device from 'core/js/device';
+import notify from 'core/js/notify';
+import ComponentView from 'core/js/views/componentView';
+import HotgridPopupView from './HotgridPopupView';
+
+class HotgridView extends ComponentView {
+
+  initialize(...args) {
+    super.initialize(...args);
+
+    this.onItemClicked = this.onItemClicked.bind(this);
+
+    this.setUpViewData();
+    this.setUpColumns();
```

Comment:
Do we need this here as the function is called in postRender ?

Reply:
Nope, unnecessary.  Fixed.